timesketch/frontend-ng/src/components/LeftPanel/AnalyserResults.vue

```diff
@@ -110,13 +109,83 @@ export default {
     meta() {
       return this.$store.state.meta
     },
-    // sketchAnalyzerList() {
-    //   return this.$store.state.sketchAnalyzerList
-    // },
+    analyzerList() {
+      return this.$store.state.sketchAnalyzerList
+    },
   },
   mounted() {
-    this.analyzerResults = this.$store.state.analyzerResults
-    this.analyzerResultsReady = true
+    let analyzerResults = []
```

Comment:
Lots of logic in the `mounted` hook. I need to play with this a bit more. I'll get back shortly